test/core/integration/concrete/lockup-tranched/streamed-amount-of/streamedAmountOf.tree

```diff
@@ -3,10 +3,7 @@ StreamedAmountOf_Lockup_Tranched_Integration_Concrete_Test
    ├── given start time in present
    │  └── it should return zero
    └── given start time in past
-      ├── given end time not in future
```

Comment:
Rationale behind removing them? Also, modifiers still exist in the corresponding test contracts.

Reply:
same reason as here

https://github.com/sablier-labs/v2-core/pull/1075#discussion_r1843568546

already tested in `SETTLED` status branch

also re this https://github.com/sablier-labs/v2-core/pull/1075#discussion_r1843598254

we should consider for `SETTLED` this modifier for both `now == end && now > end`:

```solidity
modifier runBothCases() {
    vm.warp({ newTimestamp: defaults.END_TIME() });
    _;

    vm.warp({ newTimestamp: defaults.END_TIME() + 1 });
    _;
}
```